File: RKGeonames/EconomyData.m
//
//  EconomyData.m
//  RKGeonames
//
//  Created by Stefan Burettea on 13/09/2013.
//  Copyright (c) 2013 Stefan Burettea. All rights reserved.
//

#import "EconomyData.h"

@implementation EconomyData

// |+|=======================================================================|+|
// |+|                                                                       |+|
// |+|    FUNCTION NAME: initWithCurrency                                    |+|
// |+|                                                                       |+|
// |+|                                                                       |+|
// |+|    DESCRIPTION:                                                       |+|
// |+|                                                                       |+|
// |+|                                                                       |+|
// |+|    PARAMETERS:    none                                                |+|
// |+|                                                                       |+|
// |+|                                                                       |+|
// |+|                                                                       |+|
// |+|    RETURN VALUE:  N/A                                                 |+|
// |+|                                                                       |+|
// |+|                                                                       |+|
// |+|=======================================================================|+|
- (id)initWithCurrency:(NSString *)currency
                   gdp:(NSString *)gdp
                 gdppc:(NSString *)gdppc
                 gnipc:(NSString *)gnipc;
{
    self = [super init];
    if(self)
    {
        _currency = currency;
        _gdp = gdp;
        _gdppc = gdppc;
        _gnipc = gnipc;
    }
    
    return self;
}

@end

File: RKGeonames/EconomyData.swift
//
//  EconomyData.swift
//  RKGeonames
//
//  Created by Stefan Buretea on 9/21/14.
//  Copyright (c) 2014 Stefan Burettea. All rights reserved.
//

import Foundation

class EconomyData : NSObject {
    
    let currency: NSString
    let gdp: NSString
    let gdppc: NSString
    let gnipc: NSString
    
    init(currency: NSString, gdp: NSString, gdppc: NSString, gnipc: NSString) {
        
        self.currency = currency
        self.gdp = gdp
        self.gdppc = gdppc
        self.gnipc = gnipc
    }
    
    class func data() -> EconomyData {
        
        return EconomyData(currency: "" as NSString!, gdp: "" as NSString!, gdppc: "" as NSString!, gnipc: "" as NSString!)
    }
    
}